client/src/pages/Home.js

```diff
@@ -12,8 +12,8 @@ function Home() {
   const {
     loading,
     data: { getPosts: posts }
-  } = useQuery(FETCH_POSTS_QUERY);
-
+  } = {} } = useQuery(FETCH_POSTS_QUERY);
```

Comment:
First bracket shouldn't be there. 

`const { loading, data: { getPosts: posts } = {} } = useQuery(
    FETCH_POSTS_QUERY
  );`

Reply:
you're right! I even have it your way within my own project. 
```suggestion
  = {} } = useQuery(FETCH_POSTS_QUERY);
```